### ISSUE TITLE
Prune jakarta.servlet.jsp.el.Expression, VariableResolver etc

### ISSUE THREAD

**Issue by arjantijms** [jakartaee/pages#225]
In the `jakarta.servlet.jsp.el` we have a couple of types that were deprecated more than a decade ago:

```
 @deprecated As of JSP 2.1, replaced by {@link jakarta.el.ValueExpression}
```

We unfortunately forgot to remove them for Pages 3.1/Jakarta EE 10, but should do so for the next version.

**Comment by markt-asf** [comment]
+1. That means the next version will be 4.0

**Comment by markt-asf** [comment]
Fixed as part of #234
